src/lib/webClipper/sanitizeHtml.ts

```
@@ -0,0 +1,48 @@
+    elements.forEach((el) => el.remove());
+  });
+
+  const allElements = div.querySelectorAll("*");
+  allElements.forEach((el) => {
+    const attributesToRemove: string[] = [];
+    for (const attr of el.attributes) {
+      if (attr.name.startsWith("on") || attr.name === "style") {
+        attributesToRemove.push(attr.name);
+      }
+    }
+    attributesToRemove.forEach((attr) => el.removeAttribute(attr));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)